python/lsst/pipe/tasks/processBrightStars.py

```diff
@@ -183,8 +199,8 @@ class ProcessBrightStarsTask(pipeBase.PipelineTask, pipeBase.CmdLineTask):
     def __init__(self, butler=None, initInputs=None, *args, **kwargs):
         super().__init__(*args, **kwargs)
         # Compute (model) stamp size depending on provided "buffer" value
-        self.modelStampSize = (int(self.config.stampSize[0]*self.config.modelStampBuffer),
-                               int(self.config.stampSize[1]*self.config.modelStampBuffer))
+        self.modelStampSize = [int(self.config.stampSize[0]*self.config.modelStampBuffer),
+                               int(self.config.stampSize[1]*self.config.modelStampBuffer)]
```

Comment:
I’m fine with this bug fix here and would keep it as it’s own commit, but could you split out the unrelated doc update and add a note in the commit message describing the bug that needed fixing?